.github/workflows/backend-e2e.yml

```
@@ -0,0 +1,209 @@
+  contents: read
+
+concurrency:
+  group: backend-e2e-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  NODE_VERSION: 24
+
+jobs:
+  e2e:
+    name: e2e ${{ matrix.shard }}/3 (${{ env.TIER_MODE }})
```

actionlint (1.7.12)

[error] context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)